.github/workflows/test_docker.yml

```diff
@@ -36,6 +36,8 @@ jobs:
     - name: Test
       run: |
         CONTAINER_ID="$(docker run --rm -v $(pwd):/root/playwright --name playwright-docker-test --workdir /root/playwright/ -d -t playwright-python:localbuild-focal /bin/bash)"
+        # Fix permissions for Git inside the container
+        docker exec "${CONTAINER_ID}" chown -R root:root /root/playwright
```

Comment:
Do we have the same issue in production docker images?

Reply:
no, this is only because we mount a Git repository (outside) which was created by the GitHub Actions user inside the container.